cli/src/config.scala

```diff
@@ -0,0 +1,39 @@
+import os.{Path, FilePath, RelPath}
+
+object Config {
+  lazy val defaultWorkspace = "workspace"
```

Comment:
What's the point in making it `lazy`?

Reply:
I think I was trying to make something else, so it's just a rudiment, I'll get rid of it.